src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const notification = require('./notifications')
+const { convertToCase } = require('./convertToCase');
+
+const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const { pathname, searchParams } = new URL(req.url, `http://${req.headers.host}`);
+    const toCase = searchParams.get('toCase');
+    const textToConvert = pathname.slice(1);
+    const errorNotifications = {
+      errors: [],
+    };
+
+    res.setHeader('Content-Type', 'application/json');
+
+    if (!textToConvert) {
+      errorNotifications.errors.push({
+        message: notification.textMissing,
+      });
+    }
+
+    if (!toCase) {
+      errorNotifications.errors.push({
+        message: notification.toCaseMissing,
+      });
+    }
+
+    if (toCase && !supportedCases.includes(toCase)) {
+      errorNotifications.errors.push({
+        message: notification.toCaseNotExist,
+      });
+    }
+
+    if (errorNotifications.errors.length > 0) {
+      res.statusCode = 400;
+      return res.end(JSON.stringify(errorNotifications));
```

Comment:
DRY
Create separate function for it & reuse
```suggestion
      return send(400, errorNotifications);
```

Reply:
Done